reactsite/src/reportWebVitals.js

```diff
@@ -0,0 +1,13 @@
+const reportWebVitals = onPerfEntry => {
```

Comment:
(Creating a comment mainly because I want to see how comments work...)

But also: can you say a little about what this function is doing?

Reply:
This was included in the create-react-app; I'm guessing based on names that it reports browser stats such as "time to first byte".

Reply:
Makes sense, thanks!